dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -42,7 +42,12 @@ RUN pip2 install 'setuptools==40.8.0'
 # NOTE: Only install enum34 for Python 2, as our version of Python 3 is 3.7, which
 # contains 'enum' as part of the standard library. Installing enum34 there will
 # cause conflicts.
-RUN pip2 install enum34
+RUN pip2 install enum34 \
+        grpcio==1.26.0 \
+        grpcio-tools==1.26.0
+
+RUN pip3 install grpcio==1.26.0 \
+        grpcio-tools==1.26.0
```

Comment:
>1.26.0 [](http://example.com/codeflow?start=22&length=6)

Why specify old version? any blocking issue to use latest version?

Reply:
See previous comment.